doc/user/content/overview/open-cdc.md

```diff
@@ -0,0 +1,139 @@
+
+## Example Open CDC Schema
+
+You define the Open CDC format in the Avro schema when a source is created.
+
+
+```sql
+  CREATE MATERIALIZED SOURCE name_of_source
+  FROM KAFKA BROKER 'kafka_url:9092' TOPIC 'name_of_kafka_topic'
+  FORMAT AVRO USING SCHEMA '<schema goes here>'
+  ENVELOPE MATERIALIZE
+```
+
+The following schema specifies that records will consist of `id` and `price` fields.
+
+```[
+  {
+    "type": "array",
+    "items": {
+      "type": "record",
+      "name": "update",
+      "namespace": "com.materialize.cdc",
+      "fields": [
+        {
+          "name": "data",
+          "type": {
+            "type": "record",
+            "name": "data",
+            "fields": [
+              {
+                "name": "id",
+                "type": "long"
+              },
+              {
+                "name": "price",
+                "type": [
+                  "null",
+                  "int"
+                ]
+              }
+            ]
+          }
+        },
+        {
+          "name": "time",
+          "type": "long"
+        },
+        {
+          "name": "diff",
+          "type": "long"
+        }
+      ]
+    }
+  },
+  {
+    "type": "record",
+    "name": "progress",
+    "namespace": "com.materialize.cdc",
+    "fields": [
+      {
+        "name": "lower",
+        "type": {
+          "type": "array",
+          "items": "long"
+        }
+      },
+      {
+        "name": "upper",
+        "type": {
+          "type": "array",
+          "items": "long"
+        }
+      },
+      {
+        "name": "counts",
+        "type": {
+          "type": "array",
+          "items": {
+            "type": "record",
+            "name": "counts",
+            "fields": [
+              {
+                "name": "time",
+                "type": "long"
+              },
+              {
+                "name": "count",
+                "type": "long"
+              }
+            ]
+          }
+        }
+      }
+    ]
+  }
+  ]
+  ```
+
+The updates for a single record might look like this:
+
+```
+{"array":[{"data":{"id":5,"price":{"int":10}},"time":1424849130111,"diff":1}]}
+{"array":[{"data":{"id":5,"price":{"int":12}},"time":1424849130112,"diff":1}]}
+{"array":[{"data":{"id":5,"price":{"int":12}},"time":1424849130111,"diff":-1}]}
+{"array":[{"data":{"id":5,"price":{"int":10}},"time":1424849130113,"diff":-1}]}
+```
+
+This will be summarized as:
+
+```
+{"com.materialize.cdc.progress":{"lower":[1424849130110],"upper":[1424849130116],"counts":[{"time":1424849130112,"count":1},{"time":1424849130111,"count":2}, {"time": 1424849130113, "count": 1}]}}
+```
+`lower` and `upper` define the bounds of the timestamps.
+
+Materialize should see 1 message for time 1424849130112, 2 messages for time 1424849130111, and 1 message for time 1424849130113. If there are 2 messages for time 1424849130113 with identical data updates, Materializes determines that this is a duplicate entry and discards one message.
```

Comment:
Is "message" something that the user will understand here? What about "update" instead (based on words used just above), or is this chosen on purpose (I think you are right that pre-deduplication they are still "messages" as they have not yet been accepted as updates, but that's an interesting concept to introduce).

Reply:
I changed this to "update" but also added a discussion of "progress messages", which I am not sure is the best term. @umanwizard ?